### ISSUE TITLE
Present a user-friendly error-message when a Tor endpoint is enabled, but Tor is not installed

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [vladimiry/ElectronMail#438]
Title pretty much says it all: at least on my machine, without Tor installed, selecting the 'Tor version 3 address (proton-mail)' API entry point, predictably, causes you to be unable to login.

However, instead of any sort of error-message or any indication whatsoever *why* you can't login, the webview just … freezes, offering no feedback, and disallowing the user from logging in.

I'd suggest adding a network-timeout and coherent error-message to the login-submission process!

(Beginners following the setup flow are presented "Tor" as an option without necessarily knowing what it is, or that they need to install it separately — a related-but-slightly-different task may be 'improve the onboarding process for Tor-curious newbies overall.')

**Comment by vladimiry** [comment]
The readme/project-front-page includes the https://github.com/vladimiry/ElectronMail/issues/113#issuecomment-529130116 link, see the following bullet:

> - :mailbox: **Multi accounts** support including supporting individual [API entry points](https://github.com/vladimiry/ElectronMail/issues/29). For example, you can force the specific email account added in the app connect to the email provider via the [Tor](https://www.torproject.org/) only by selecting the `Tor version 3 address` API entry point in the dropdown list and configuring a proxy as described in [this](https://github.com/vladimiry/ElectronMail/issues/113#issuecomment-529130116) message.

That's Windows-specific but could also be translated to macOS by advanced users. Of course I agree that it's not enough in terms of UX, ie the UX could be improved although I generally believe that those who use Tor normally understand what that thing is.

**Comment by ELLIOTTCABLE** [comment]
Yep! As with all these issues i'm opening, these are low-ish priority, polish, user-experience things. Hopefully such are welcome in your triage process. <3

**Comment by vladimiry** [comment]
> Hopefully such are welcome in your triage process. <3

Yep, I like to paint the labels, not the same thing as handling the actual issue though. I don't normally specify the priority here, as, you know, "not a product".
